### DISCUSSION TITLE
Configuration Issue?

### DISCUSSION THREAD

**Discussion by Frank482** [arendst/Tasmota#18111] [Q&A]
I installed PIO Home - Tasmota-development - Visual Studio Code and Tasmota following
https://tasmota.github.io/docs/Visual-Studio-Code/

Everything works fine including upload. Tasmota terminal via webbrowser shows the correct build version in "Information".
But when I change to another environment in PIO I get the message:
"Running the contributed command: 'platformio-ide.switchProjectEnv' failed".
Tasmota again shows the new version correctly.

Is this an issue?

**Comment by Jason2866** [comment]
What exactly do you do to change to an other environment?
Two ways are possible. Change in `platformio override.ini` the entry(s) for the used env with changing the entry under `default_envs` or use Platformio IDE VSC menu to select the used env

Never got the error message you mentioned.
